src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -375,9 +375,10 @@ public IActionResult OrganisationName(string post)
             return View("OrganisationName", organisationVm);
         }
 
+        [Route("TotalEmployees/{id}")]
         [Route("TotalEmployees")]
         [Authorize(Policy = PolicyNames.CanEditReport)]
-        public IActionResult TotalEmployees()
+        public IActionResult TotalEmployees(bool? id)
```

Comment:
true/false should not be part of the route. It could be an optional query param?

Also this should be renamed. It is not an id and the name id is confusing and misleading